src/ApiTestCase.php

```diff
@@ -228,37 +214,100 @@ private function getCalledClassFolder()
         $calledClass =  get_called_class();
         $calledClassFolder = dirname((new \ReflectionClass($calledClass))->getFileName());
 
-        $this->folderExists($calledClassFolder);
+        $this->directoryCheck($calledClassFolder);
 
         return $calledClassFolder;
     }
 
+    protected function loadAllFixtures()
+    {
+        $this->loadFixturesByName();
+    }
+
+    protected function loadFixturesByName(array $fixturesNames = array())
+    {
+        $loader = new Loader();
+
+        $directory = $this->getFixturesFolder();
+        $this->directoryCheck($directory);
+        $paths = $this->getFixturePathsFromDirectory($directory, $fixturesNames);
+
+        foreach ($paths as $path) {
+            $objects = $loader->load($path);
+            $this->persistObjects($objects);
+        }
+
+        $this->entityManager->flush();
+    }
+
     /**
      * @param string $directory
+     * @param array $fixtureNames
      *
      * @return array
      */
-    private function getFilesFromDirectory($directory)
+    private function getFixturePathsFromDirectory($directory, array $fixtureNames = array())
```

Comment:
We may use `\RecursiveDirectoryInterator`, it'll make it cleaner and (most probably) faster (and will result in an iterator of handy `\SplFileInfo`).